modules/ROOT/pages/tutorial-services.adoc

```diff
@@ -24,10 +26,11 @@ We will store this script into `/usr/local/bin/public-ipv4.sh` when we provision
 
 == Writing the systemd service
 
-We need to call the script from the previous section by using a systemd unit. Here is one that works for what we want, which is to execute on first boot and not again:
+We need to call the script from the previous section by using a systemd unit. Let's write a systemd unit into the `issuegen-public-ipv4.service` file that does what we want, which is to execute on first boot and not again:
 
 [source,service]
 ----
+cat <<'EOF' > issuegen-public-ipv4.service
```

Comment:
This source block is still declared as type `service` but with the new shell redirects to write the output to a file it should probably be a `bash` block instead?

Reply:
fixed!